composer.json

```diff
@@ -18,7 +18,7 @@
         }
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=7.0",
```

Comment:
I'd rather like having the same version for everyone, what about `"php": "7.0"`?